index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="bar-container">
+        <nav>
+           <img src="/Images/Light.png"/>
+           <img src="/Images/app_bar.png"/>
+        </nav>
+   </header>
+   <div class="container">
+       <section class="head">
+           <h2> 
+               Hey there. I’m Santiago. 
+               <div class="title"> I’m a software developer </div>
+           </h2>
+        </section>
+        <section class="text">
+            <p>
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.
+            </p>
+        </section>
+        <div class="socials">
+            <ul>
+                <li> <a href="https://twitter.com/NoizeSalt"> <img src="Images/twitter.png" alt="Twitter"/> </a> </li>
+                <li> <a href="https://www.linkedin.com/in/santiago-ruido-a1404880/"> <img src="Images/Linkedin icon.png" alt="LinkedIn"/> </a> <li>
+                <li> <a href="https://medium.com/"> <img src="Images/medium.png" alt="Medium"/> </a> </li>
+                <li> <a href="https://github.com/Arch-Noize"> <img src="Images/github.png" alt="GitHub"/> </a> </li>
+                <li> <a href="https://www.angellist.com/"> <img src="Images/angellist.png" alt="Angellist"/> </a> </li>
+             </ul>
```

Comment:
[OPTIONAL]

- Your social media icons are currently not displaying. Kindly ensure to use the correct image paths so they can be visible on the UI
![image](https://github.com/Arch-Noize/setup-and-mobile-first/assets/63473584/f8683740-df9e-4803-a0a2-44c0e49761c1)